Versions-mac.plist.in

```diff
@@ -18,6 +18,7 @@
 			<string>10.14</string>
 			<string>10.15</string>
 			<string>10.16</string>
```

Comment:
Do we want to remove `10.16` here?

Reply:
part of https://github.com/xamarin/xamarin-macios/issues/8943
My guess is yes, but I was not sure this bump was the best place to do it :)